akka-docs/build.sbt

```diff
@@ -22,8 +22,4 @@ paradoxProperties ++= Map(
   "snip.akka.base_dir" -> ((baseDirectory in Test).value / "..").getAbsolutePath
 )
 
-resolvers += Resolver.bintrayRepo("2m", "maven")
-paradoxTheme := Some("com.lightbend.akka" % "paradox-theme-akka" % "b74885b8+20170511-1711")
-paradoxNavigationDepth := 1
-paradoxNavigationExpandDepth := Some(1)
-paradoxNavigationIncludeHeaders := true
```

Comment:
They are part of sbt-paradox-akka now: https://github.com/akka/akka-paradox/commit/33fe4a724e4ff98e6440395bc6797b6ff28d935a#diff-cd79816a71bd31caf7a78b4ec42412ceR20